hyperspy/tests/signal/test_fourier_transform.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2007-2016 The Hyperspy developers
+#
+# This file is part of  Hyperspy.
+#
+#  Hyperspy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  Hyperspy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  Hyperspy.  If not, see <http://www.gnu.org/licenses/>.
+
+import nose.tools as nt
+import numpy as np
+from numpy.testing import assert_allclose
+
+from hyperspy.signals import Spectrum, Image, Signal
+
+
+class TestImageFFT():
+    def setUp(self):
+        im = Image(np.random.random((2, 3, 4, 5)))
+        self.signal = im
+
+    def test_fft_ifft(self):
+        im = self.signal
+        im_fft = im.fft()
+        im_ifft = im_fft.ifft()
+        nt.assert_true(isinstance(im_ifft, Signal))
+        assert_allclose(im.data,  im_ifft.data, atol=1e-3)
+
+        im_fft = im[0].fft()
```

Comment:
Ouh thanks for that - this change kind of went over my head because the only way I learnt to do it was as above.... So the correct usage should be that when we want to select an a single signal from a stack of signals we use inav?